aiohttp/abc.py

```diff
@@ -117,11 +119,23 @@ def __await__(self) -> Generator[Any, None, StreamResponse]:
         """Execute the view handler."""
 
 
+class ResolveResult(TypedDict):
```

Comment:
add a docstring?

Reply:
Good call. added

Reply:
Perhaps, we should also link it from `abc.rst`?

Reply:
It looks like the `AbstractResolver` isn't mentioned at all in `abc.rst`

Reply:
I added it to `abc.rst`